src/mantine-core/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import cx from 'clsx';
+import { DefaultProps, mergeStyles, useMantineTheme } from '../../theme';
+import useStyles from './Pagination.styles';
+import { usePagination, PaginationParams } from './usePagination/usePagination';
+import { DefaultItem, PaginationItemProps } from './DefaultItem/DefaultItem';
+
+export type { PaginationItemProps } from './DefaultItem/DefaultItem';
+
+export type PaginationStylesNames = keyof ReturnType<typeof useStyles>;
+
+export interface PaginationProps extends DefaultProps<PaginationStylesNames>, PaginationParams {
+  /** Change item component  */
+  itemComponent?: React.FC<PaginationItemProps>
+
+  /** Button color from theme */
+  color?: string;
+
+  /** Page selected on initial render */
+  initialPage?: number;
+
+  /** Controlled active page number */
+  page?: number;
+
+  /** Total amount of pages */
+  total: number;
+
+  /** Siblings amount on left/right side of selected page */
+  siblings?: number;
+
+  /** Amount of elements visible on left/right edges */
+  boundary?: number;
+
+  /** Callback fired after change of each page */
+  onChange?: (page: number) => void;
+}
+
+export function Pagination({
+  itemComponent: Item = DefaultItem,
+  className,
+  classNames,
+  style,
+  styles,
+  themeOverride,
+  page,
+  initialPage = 1,
+  color,
+  total,
+  siblings = 1,
+  boundary = 1,
+  onChange,
+  ...others
+}: PaginationProps) {
+  const theme = useMantineTheme(themeOverride);
+  const classes = useStyles(
+    { theme, color },
+    classNames,
+    'pagination'
+  );
+  const _styles = mergeStyles(classes, styles);
+
+  const {
+    paginationRange,
+    goToPage,
+    goNextPage,
+    goPrevPage,
+    activePage,
+  } = usePagination({
+    page,
+    siblings,
+    total,
+    onChange,
+    initialPage,
+    boundary,
+  });
+
+  return (
+    <nav className={cx(classes.root, className)} style={{ ...style, ..._styles.root }} {...others}>
+      <Item
+        page="prev"
+        onClick={goPrevPage}
+        aria-label="go to previous page"
```

Comment:
Do not hardcode labels, it cannot be translated or changed this way

Reply:
That's why I've left the proposal for the `getItemAriaLabel ` prop. A11y leaves a warning if I will not leave some info about the button. I can delete it but that will cause some warnings in console after `npm test`

Reply:
@rtivital I'm doing fixes to PR. What should I do here?

Reply:
Yep, `getItemAriaLabel` seems like a good solution, do not forget to include it to Accessibility docs section

Reply:
Sure